snakefile.py

```diff
@@ -172,13 +199,12 @@ def tool(name):
     log: os.path.join(LOG_DIR, 'bwa_index.log')
     shell: """mkdir -p {INDEX_DIR};
 ln -sf {input} {INDEX_DIR};
-cd {INDEX_DIR};
```

Comment:
hmm, are you sure this can be removed?  bwa operates on the current directory, so your tests may only have worked accidentally.

Reply:
hmm...I'm usually not within the test directory when I run the tests. 
I thought having cd within the code could be a vulnerability issue?

But ok, if you say that this one was there on purpose I could put it back.